src/category_theory/monad/adjunction.lean

```diff
@@ -148,6 +187,80 @@ from `C` to the category of Eilenberg-Moore algebras for the adjunction is an eq
 class comonadic_left_adjoint (L : C ⥤ D) extends is_left_adjoint L :=
 (eqv : is_equivalence (comonad.comparison (adjunction.of_left_adjoint L)))
 
+attribute [instance] monadic_right_adjoint.eqv comonadic_left_adjoint.eqv
+
+/-- Given an adjunction `L ⊣ R₁` and an isomorphism `R₁ ≅ R₂` the monads induced on `C` by `R₁` and
+`R₂` are isomorphic. -/
+@[simps]
+def to_monad_iso_of_nat_iso_right {L : C ⥤ D} {R₁ R₂ : D ⥤ C} (h₁ : L ⊣ R₁) (i : R₁ ≅ R₂) :
```

Comment:
Do we have `to_monad_iso_of_nat_iso_left` already? If not, then I think we should probably add it at this point?

Reply:
We don't have it. I'll have a go at making it!